bucket/apimtemplate.json

```diff
@@ -1,19 +1,26 @@
 {
-    "version": "0.5",
+    "version": "0.7",
     "description": "Azure API Management DevOps Resource Kit",
     "homepage": "https://github.com/Azure/azure-api-management-devops-resource-kit",
     "license": "MIT",
     "depends": "azure-cli",
```

Comment:
Should we change it to a suggestion instead?

Future PR of course

Reply:
Yes I will do that

Reply:
@Valinor for this should we instead create a new PR labelled good first issue?

Would help new Scoop devs who wish to participate but haven't participate in the development and if 2 weeks later no PR is open, we manually update it?